tree/dataframe/test/dataframe_simple.cxx

```diff
@@ -513,6 +513,26 @@ TEST_P(RDFSimpleTests, BookCustomAction)
    EXPECT_EQ(*maxSlot, nWorkers-1);
 }
 
+TEST(RDFSimpleTests, StandardDeviation)
+{
+   RDataFrame rd1(8);
+   int i(0);
+   auto stdDev = rd1.Define("b1",
+                            [&i]() {
+                               ++i;
+                               return i;
+                            })
+                    .StdDeviation<int>("b1");
```

Comment:
please also add a test for `StdDeviation("b1")`, i.e. no template parameter and therefore jitting